mobly/config_parser.py

```diff
@@ -80,15 +80,15 @@ def _validate_testbed_configs(testbed_configs):
         _validate_testbed_name(name)
         # Test bed names should be unique.
         if name in seen_names:
-            raise MoblyConfigError("Duplicate testbed name %s found." % name)
+            raise MoblyConfigError('Duplicate testbed name %s found.' % name)
         seen_names.add(name)
 
 
 def _verify_test_class_name(test_cls_name):
```

Comment:
I don't see any reason to enforce a test class naming convention. Any advantages?